### ISSUE TITLE
Null pointer when converting OData -> Open API

### ISSUE THREAD

**Issue by dmchughaxway** [Axway-API-Management-Plus/apim-cli#399]
### APIM-CLI version

1.13.5

### API-Management version

7.7.20221130

### Bug description

Steps to reproduce

### Steps to reproduce

- Download the edmx file from the SAP - https://api.sap.com/api/OP_HANDLINGUNIT_0001/overview
- Using the edmx file attempt to create an api via the CLI i.e. apim api import 

### Relevant log output

```shell
2023-06-22 10:11:33,873 [ODataV4Specification] ERROR: Error parsing OData V4 MetaData.
java.lang.NullPointerException
	at com.axway.apim.api.specification.ODataV4Specification.getSchemaForType(ODataV4Specification.java:505)
	at com.axway.apim.api.specification.ODataV4Specification.getSchemaForType(ODataV4Specification.java:504)
	at com.axway.apim.api.specification.ODataV4Specification.getSchemaForType(ODataV4Specification.java:468)
	at com.axway.apim.api.specification.ODataV4Specification.getPathItemForEntity(ODataV4Specification.java:318)
	at com.axway.apim.api.specification.ODataV4Specification.parse(ODataV4Specification.java:79)
	at com.axway.apim.api.specification.APISpecificationFactory.getAPISpecification(APISpecificationFactory.java:66)
	at com.axway.apim.api.specification.APISpecificationFactory.getAPISpecification(APISpecificationFactory.java:43)
	at com.axway.apim.apiimport.APIImportConfigAdapter.addAPISpecification(APIImportConfigAdapter.java:201)
	at com.axway.apim.apiimport.APIImportConfigAdapter.getDesiredAPI(APIImportConfigAdapter.java:154)
	at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:68)
	at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:53)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.axway.apim.cli.APIManagerCLI.run(APIManagerCLI.java:133)
	at com.axway.apim.cli.APIManagerCLI.main(APIManagerCLI.java:77)
2023-06-22 10:11:33,874 [APISpecificationFactory] ERROR: API: OData v4 Handling Unit has a unknown/invalid API-Specification: <?xml version="1.0" encoding="utf-8"?><edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" xmlns="http://docs.oasis-open.org/odata/ns/edm"><edmx:Reference Uri="/sap/opu/odata...
2023-06-22 10:11:33,876 [APIImportApp] INFO : You may enable debug to get more details. See: https://github.com/Axway-API-Management-Plus/apim-cli/wiki/9.1.-Enable-Debug
2023-06-22 10:11:33,876 [APIImportApp] ERROR: Can't handle API specification. No suitable API-Specification implementation available.
```

**Comment by rathnapandi** [comment]
Hi @dmchughaxway, 

I have fixed the exception, Can you test with
[axway-apimcli-1.14.1-SNAPSHOT.tar.gz](https://github.com/Axway-API-Management-Plus/apim-cli/files/11986836/axway-apimcli-1.14.1-SNAPSHOT.tar.gz)?

Thanks
Rathna

**Comment by rathnapandi** [comment]
fixed in v1.14.1
